### ISSUE TITLE
Make Extension Responsive (RWD theme)

### ISSUE THREAD

**Issue by IvanChepurnyi** [EcomDev/checkitout#2]
I need a frontend developer to create a new layout as described in the following blog post:
http://www.ecomdev.org/2012/11/29/magento-checkout-extension-ideas-architecture-revealed.html (Layout Handles section)

Please comment this issue with your intention to help, and I will add you as a collaborator on the project.

**Comment by bluec** [comment]
@IvanChepurnyi is this still an active project? I may be prepared to put a small bounty forward for someone to create a RWD layout for you.

**Comment by andrewhowdencom** [comment]
I am using it for some work now; active-ish

I heavily customise the templates when I implement it (or have done, the last two implementations)

**Comment by bluec** [comment]
Yes I'm using it too and love the ease of customisation and clean code compared to other solutions. It would be nice if there was an rwd theme included and I expect various people have already done their own rwd themes so it would be good to get one contributed back into the project.

If anyone out there has done one or would like to do one then please respond to this!
